native/explorer/src/series.rs

```diff
@@ -1210,3 +1212,42 @@ pub fn s_atan(s: ExSeries) -> Result<ExSeries, ExplorerError> {
     let s1 = s.f64()?.apply(|o| o.atan()).into();
     Ok(ExSeries::new(s1))
 }
+
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn s_to_arrow(env: Env, s: ExSeries) -> Result<Binary, ExplorerError> {
+    let res = &s.resource;
+    let s = s.rechunk();
+    let array = s.to_arrow(0);
+    let c_array = Box::new(ffi::export_array_to_c(array));
+    let array_ptr: *const ffi::ArrowArray = &*c_array;
+
+    // https://stackoverflow.com/questions/28127165/how-to-convert-struct-to-u8
+    let transmuted: &[u8] =
+        unsafe { slice::from_raw_parts(array_ptr as *const u8, mem::size_of::<ffi::ArrowArray>()) };
+
+    let bin = unsafe { res.make_binary_unsafe(env, |_| transmuted) };
+
+    // We tell the compiler to forget about this box, and therefore don't
+    // call the destructor (drop). This will keep the array in memory, without
+    // causing a segfault. But it's probably considered a leak.
+    // See: https://doc.rust-lang.org/std/mem/fn.forget.html
+    mem::forget(c_array);
```

Comment:
I am assuming `c_array` is freshly allocated by this function and it is not a reference to the Series. If so, this is 100% a leak. In this case, we have two options:

* Copy the binary

* If there is a `ffi` function that tells us the byte size of the Arrow Array, then we can ask Erlang to allocate a NewBinary and have the `ffi::export_array_to_c` write to this new binary